src/network/networkd-address-generation.c

```diff
@@ -359,8 +359,7 @@ int config_parse_address_generation_type(
                                 addr = addr_alloc;
                         }
                 } else if (*addr == ',') {
```

Comment:
remove brackets?

Reply:
Done. I wasn't sure if it wouldn't look too weird, but I guess it could be worse.